sanskrit_parser/util/normalization.py

```diff
@@ -12,9 +12,10 @@
 logger = logging.getLogger(__name__)
 
 deletions = re.compile("[\u200b\u200c\u200d,'-;().?!\"0123456789{}#\r]")
+rakarantas = ['punaH', 'antaH']
 
 
-def normalize(s):
+def normalize(s, vis_end=True):
```

Comment:
Change to "replace_vis_end" to make it clear that it will replace visargas

Reply:
Changed to replace_ending_visarga, and strict_io_replace_ending_visarga in normalization and sanskrit_base respectively. This makes things clearer